srcpkgs/autotiling/template

```diff
@@ -9,7 +9,8 @@ short_desc="Automatically switch horizontal/vertical split orientation in i3/swa
 maintainer="Adrian Göransson <adriangoransson@gmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/nwg-piotr/autotiling"
+changelog="https://raw.githubusercontent.com/nwg-piotr/autotiling/master/CHANGELOG.md"
```

Comment:
last updated 2 years ago (before 1.5 was released, probably no worth adding

Reply:
I added it as insurance against being asked to add it. Guess that didn't work out.